mne_bids/utils.py

```diff
@@ -503,6 +503,28 @@ def _get_brainvision_paths(vhdr_path):
     return (eeg_file_path, vmrk_file_path)
 
 
+def copyfile_ctf(src, dest):
+    """Copy and rename CTF files to a new location.
+
+    Parameters
+    ----------
+    src, dest : str
```

Comment:
This is not how we document arguments. You should follow the numpy docstring convention. Each argument should have a separate line